### PR TITLE
Sass for util components and terms page

### LINKED ISSUE #419: Refactoring Terms & Util component
Refactoring the `Terms` and `Util` as per the [proposed style](https://github.com/HITK-TECH-Community/Community-Website/blob/main/frontend/PROPOSAL.md). 

Note- Please make sure to follow all the guidelines mentioned in the proposed style properly!

### PR DESCRIPTION
**Pull request by ashuvssut**
## Issue that this pull request solves

 Closes #419 

------------------
_--I have participated in DWoC and SWoC_

### CODE REVIEW

File: frontend/src/components/Button/Button.jsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const Button3 = (props) => {
   return (
     <button
       type={props?.type || "button"}
-      className="main-btn main-btn-2 main-btn-3"
+      className={`${style["main-btn"]} ${style["main-btn-2"]} ${style["main-btn-3"]}`}
```
Comment (ashuvssut) [inline frontend/src/components/Button/Button.jsx:47]: before: `className="main-btn main-btn-2 main-btn-3"`

after: `className={`${style["main-btn"]} ${style["main-btn-2"]} ${style["main-btn-3"]}`}`

in this line, if you can suggest a better clean way to write multiple css classes to an element, then I ll be glad to implement that.

**Review by salil-naik — APPROVED** [review]
Amazing job @ashuvssut 👌

**Comment by salil-naik** [comment]
@Kajol-Kumari this PR follows the new folder structure. There was no `util` folder in the proposal. And hence we have conflicts. Should we retain the util folder? Or remove it and put `button` and `scroll-to-top` components in the `components` folder directly?

**Comment by ashuvssut** [comment]
buttons, scroll-to-top are all reusable components that are used by all pages. so I thought keeping it in components folder is the right choice. so i deleted the utils folder😅😅

**Review by AnjaliPatle — CHANGES_REQUESTED** [review]
Hey @ashuvssut, great job! Just some small changes:
Instead of creating new folders for Button, Dropdown and ScrollToTop in `src/components` directly, please retain the util folder and add it under `src/components` with all other folders inside it.

**Comment by ashuvssut** [comment]
new one must be like:-
`components/`--|
                          |--`util/`--|
                          |              |--`button/`
                          |              |--`scrollToTop/`
                          |              |--`dropdown/`


instead of current one which is like:-
`components/`--|
                          |
                          |--`button/`
                          |--`scrollToTop/`
                          |--`dropdown/`

Am I right? Is this the requirement?

**Comment by AnjaliPatle** [comment]
Yes @ashuvssut , you are right.

**Comment by salil-naik** [comment]
@ashuvssut you requested for review but nothing has been changed since I last reviewed. 😅  Maybe you forgot to push your changes?

**Comment by ashuvssut** [comment]
@salil-naik, sorry for that😅😅

**Comment by Kajol-Kumari** [comment]
@ashuvssut please resolve the conflict here

**Comment by ashuvssut** [comment]
@Kajol-Kumari I resolved the 2nd merge conflict. Please review

**Review by Kajol-Kumari — APPROVED** [review]
Thanks for incorporating the suggested changes @ashuvssut 
LGTM :)

**Pull request merged**